package.json

```diff
@@ -83,6 +83,7 @@
     "rollup-plugin-vue": "3.0.0",
     "rollup-watch": "4.3.1",
     "vue": "2.5.17",
+    "vue-autosuggest": "^1.5.0",
```

Comment:
Dependency should be exact.

Reply:
Will change those in another PR, they are behind in most :)